spec/system/notifications/notifications_page_spec.rb

```diff
@@ -42,8 +42,8 @@ def validate_reply(id)
     Notification.send_new_follower_notification_without_delay(follow, "Published")
     visit "/notifications"
     expect(page).to have_css("div.single-notification")
-    click_button("+ FOLLOW BACK")
-    expect(page).to have_text("FOLLOWING")
+    click_button("Follow back")
```

Comment:
Do we want to keep the `+` or is it being omitted on purpose for the new design changes

Reply:
Same here